extended/src/main/java/io/kubernetes/client/workqueue/RateLimiter.java

```diff
@@ -0,0 +1,30 @@
+package io.kubernetes.client.workqueue;
+
+import java.time.Duration;
+
+public interface RateLimiter {
```

Comment:
There are lots of other retry libraries out there (e.g. https://github.com/jhalterman/failsafe) do we need to implement this ourselves?

Reply:
didn't notice this before, will take a deep look